test/kixi/nhs/board_report_test.clj

```diff
@@ -0,0 +1,43 @@
+(ns kixi.nhs.board-report-test
+  (:use clojure.test)
+  (:require [kixi.nhs.board-report :as board-report]))
+
+
+(deftest filter-and-enrich-dataset-test
+  (testing "Testing filtering and enriching dataset")
+  ; Checks if the sequence is returned with "Year", "Indicator id" and "Value"
+  ; and that the filter works properly.
+  (is (= '({"Year" "2013/14" "Indicator id" "22" "Value" "85.7"} {"Year" "2012/13" "Indicator id" "22" "Value" "86.7"})
```

Comment:
why quoted lists rather than vectors. None of this appears to be lazy (unless I missed something)